api/lib/validation/schemaValidator.js

```diff
@@ -57,7 +57,7 @@ var getDaysInMonthInISOFormat = function() {
 };
 
 var getMetricTypes = function() {
-  var metricTypeEnum = ['memoryused'];
+  var metricTypeEnum = ['memoryutil', 'responsetime', 'throughput'];
```

Comment:
No, we can get quota for both polling and streaming, so we can just replace memoryused with memoryutil.